apps/publications/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,30 @@
+<mat-toolbar [ngStyle]="{'background': bgColor}" class="sticky-top header-container" style="display: flex; justify-content: space-between;">
+  <div style=" display: flex; flex-direction: row;">
+    <button (click)="sideNav.toggle()" [ngStyle]="{'color': iconColor}" mat-icon-button>
+      <mat-icon>menu</mat-icon>
+    </button>
+
+    <a class="ml-3" [innerHTML]="logo" routerLink="/all-publications"></a>
+    <mat-icon svgIcon="perun-publications-white" style="height: 3rem; width: 3rem"></mat-icon>
```

Comment:
The publication logo can be also a link
also please remember that not every user will have the rights to acces /all-publications, for some of the users main page can be /my-publications